src/OpenTelemetry/Metrics/MetricAggregators/SumMetricAggregatorLong.cs

```diff
@@ -43,14 +47,21 @@ public void Update<T>(T value)
                 {
                     // TODO: Confirm this doesn't cause boxing.
                     var val = (long)(object)value;
-                    if (val < 0)
+                    if (val < 0 && this.isMonotonicValue)
                     {
                         // TODO: log?
                         // Also, this validation can be done in earlier stage.
                     }
                     else
                     {
-                        this.sumLong += val;
+                        if (this.isDeltaValue)
```

Comment:
without corresponding changes in the Collect() as well, we wont get cumulative/delta outputs.

Reply:
That part was being addressed in 2173. But I guess we're abandoning that and moving it into this PR.